codegen/module_system.go

```diff
@@ -1176,6 +1207,20 @@ func GenerateInitializer(
 	)
 }
 
+// GenerateMockInitializer is like GenerateInitializer but with leaf nodes being mocks.
+func GenerateMockInitializer(
```

Comment:
nit: The existing code doesn't have a function per template, you should stick with existing style.